update all (#11)
NIFCLOUD-mbaas/monaca_data_registration: README_en.md

# 【Monaca】 Let's try NIFCLOUD mobile backend!
![Image1](/readme-img/001_en.png)

<!-- PJ Update 2020/05/07 -->
<!-- JS SDK Ver. 3.0.2 -->

## Overview
* This is a sample application that uses Monaca to register data to [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en)
<!-- * This is a sample application that uses Monaca to register data to [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/) -->

> * When you tap "Start Demo" button, data will be registered to the cloud ★

Author: `NIFCLOUD Mobile backend cloud.`
→ `NIFCLOUD mobile backend cloud.` と修正しました。

> * By this simple operation, you can experience the  [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en)

Author: 修正しました。

<!-- * By this simple operation, you can experience the  [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/) -->

> * Remember to switch language to English when you first access the control panel.

Author: 修正しました。

> This time, I will experience the data store feature.

Author: 修正しました。

> This time, I will experience the data store feature.

Author: 修正しました。

## Procedure
### 1. Sign up for [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en) and login → Create application
<!-- ### 1. Sign up for [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/) and login → Create application -->

* Access the following URL.
  * https://console.mbaas.nifcloud.com/signup
* Click "Language" in the upper right corner and select "English".

![AccountRegisterImage0](/readme-img/account_register_001.png)

> * Sign up using your Facebook, Twitter, or Google social networking account. Click the SNS account you want to use.

Author: 修正しました。

![AccountRegisterImage1](/readme-img/account_register_002.png)

* After that, the registration screen of each SNS will be displayed, so please follow the instructions.

> * Your account will be created if you set and confirm the registered email address and agree to the terms of use

Author: 修正しました。

> * The "Create new App" screen will appear, then you create the application.

Author: 修正しました。

![Image3](/readme-img/003_en.png)

> * When the application is created, the screen will look like the below figure.

Author: 修正しました。

> * These two kinds of API keys (application key and client key) are used to link the [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/) to the iOS application that created by Xcode.

Author: 修正しました。

### 2. Import project to Monaca and launch application

> * Log in to [Monaca](https://ja.monaca.io/)

Author: ありがとうございます。2か所ありましたので、いずれも修正しました。

* Click "Import" button at the top left.
* Click "Import form URL".

![Image6_01](/readme-img/006_en_01.png)

* Copy and paste the following link and Click "Next" button.
  * Project: `https://github.com/NIFCLOUD-mbaas/monaca_data_registration/archive/master.zip`
* Enter "Project Name".
  * for example: `DBDEMO`

> * When click "Import Project" button, the imported project will be created.

Reviewer: ```suggestion
* After click "Import Project" button, the imported project will be created.
```

Author: 修正しました。

![Image6_02](/readme-img/006_en_02.png)

* Select the created project and click "Open in Cloud IDE" displayed on the right.

![Image6_03](/readme-img/006_en_03.png)

* The project is opened and the preview screen is displayed

> * Let's play with the preview screen or [Monaca debugger](https://ja.monaca.io/debugger.html)!

Reviewer: ```suggestion
* Let's use the preview screen or [Monaca debugger](https://monaca.io/debugger.html) to test.
```

Author: 修正しました。

※ We recommend using preview screen/Monaca with __iPhone 6__ or higher for operation confirmation.

### 3. Setting the API key

* When the project is opened, edit the index.html

> * Paste the API key you confirmed on the management screen of [NIFCloud mobile backend](https://mbaas.nifcloud.com/en) earlier

Reviewer: ```suggestion
* Paste the API key you confirmed on the control panel screen of [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en) earlier.
```

Author: 修正しました。
（管理画面を英語にすると「control panel」なんですね。）

* If an error occurs, you can confirm the error content from [here](https://mbaas.nifcloud.com/doc/current/rest/common/error.html)
![Image1](/readme-img/001_en.png)

> * If saving successfully, let's check "Datastore" from the management screen of [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en)!

Reviewer: ```suggestion
* After saving successfully, let's check "Datastore" from the management screen of [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/en) to see registered data.
```

Author: 修正しました。

<!-- * If saving successfully, let's check "Datastore" from the management screen of [NIFCLOUD mobile backend](https://mbaas.nifcloud.com/)! -->
* `TestClass` that used for saving is created, and you can confirm data in it

> ## Explanation

Reviewer: ```Introduction of contents already implemented in sample project```
->
```Most of features are already implemented in sample project, which we will display details following.```

Author: 修正しました。

    testClass.set(key, value);
    testClass.save()
        .then(function() {
            // Save success.
            alert("New object created with objectId: " + testClass.objectId);
        })
        .catch(function(error) {
            // Save failed.
            alert("Failed to create new object, with error code: " + error.text);

>         });

Reviewer: ```If the SDK is not up to date, please refer to the image below and update your SDK yourself```
->
```
If the SDK(ncmb) is not up to date, please refer to the image below and update your SDK yourself.
Remember to remove installed SDK(ncmb) before reinstall a new one.
```

Author: こちらですが、先に提案いただいた内容を踏まえ、構成を少し変更したため、いただいた内容で修正していません。お手数ですが、修正後再度レビューをお願いします。

Reviewer: 構成変更について、承知しました！問題ないことを確認しました
